### PR TITLE
Product Linear Placement template update

### LINKED ISSUE #233: Review Product Linear Placement Template
## Description & Tasks
Review of Product Linear Placement is still correct and relevent in light of chagnes to alignment durinf deployment

- [x] Review Product Linear Placement Template
- [x] Update Documentation, if Needed
- [x] Updated Template, if Needed

### Discussion Points

1. Still correct in relation to Alignment Changes as created at IFC4.2?

### Links
[Miro Element](https://miro.com/app/board/o9J_l5rJCLk=/?moveToWidget=3074457362574420150&cot=14)
[Miro Task Card](https://miro.com/app/board/o9J_l5rJCLk=/?moveToWidget=3074457362911087570&cot=14)

### Decisions 
1. No Issues found,

### PR DESCRIPTION
**Pull request by larswik**
Fixes #233
Updated documentation (please review) and added ruleId:s for "RelativePlacement", "PlacesObject" and "LinearPositioningElement.Name" thanks to @pjanck.

### CODE REVIEW

File: IFC4x3/Templates/Product Shape/Product Placement/Product Linear Placement/Documentation.md [hunk 1]
```diff
@@ -1 +1 @@
...
+A Product Linear Placement takes into account the ISO 19148 Linear referencing standard. Linear placement is defined by a position relative to a _IfcLinearPositioningElement_ (distance along and optionally lateral, vertical and longitudinal offsets), a horizontal reference direction, and a vertical axis direction. Each product placed with Product Linear Placement has an absolute placement on the _IfcLinearPositioningElement_._Representation_ where the _IfcLinearPositioningElement_  is typically instantiated as an IfcAlignment. Therefore, similarly to how _IfcSpatialElement_._ObjectPlacement_ sets the context for all contained elements, the IfcLinearPositioningElement sets the context for all elements positioned on it. Consequently, each product placement that uses Product Linear Placement references an _IfcLocalPlacement_ which is the placement of an _IfcLinearPositioningElement_ through _IfcLinearPlacement_._PlacementRelTo_.
```
Comment (pjanck) [inline IFC4x3/Templates/Product Shape/Product Placement/Product Linear Placement/Documentation.md:1]:
```suggestion
A Product Linear Placement takes into account the ISO 19148 Linear referencing standard. Linear placement is defined by a position relative to a _IfcLinearPositioningElement_ (distance along and optionally lateral, vertical and longitudinal offsets), a horizontal reference direction, and a vertical axis direction. Each product placed with Product Linear Placement has an absolute placement on the _IfcLinearPositioningElement_._Representation_ where the _IfcLinearPositioningElement_  is typically instantiated as an IfcAlignment. Therefore, similarly to how _IfcSpatialElement_._ObjectPlacement_ sets the context for all contained elements, the _IfcLinearPositioningElement_._ObjectPlacement_ sets the context for all elements positioned on it. Consequently, each product placement that uses Product Linear Placement references the _IfcObjectPlacement_ of the _IfcLinearPositioningElement_ through _IfcLinearPlacement_._PlacementRelTo_.
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Improvement proposal.

**Review by pjanck — APPROVED** [review]
lftm

**Review by AlexBrad1eyCT — APPROVED** [review]
LFTM

**Review by czapplitec — APPROVED** [review]

**Pull request merged**